source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -232,13 +339,110 @@
         'parameters': [
             {
                 'direction': 'in',
-                'name': 'task',
+	            'name': 'task',
                 'type': 'TaskHandle'
             }
         ],
         'returns': 'int32'
     },
     'WriteAnalogF64': {
+        'parameters': [
+	        {
+                'direction': 'in',
+                'name': 'task',
+                'type': 'TaskHandle'
+            },
+            {
+                'direction': 'in',
+                'name': 'numSampsPerChan',
+                'type': 'int32'
+            },
+            {
+                'direction': 'in',
+                'name': 'autoStart',
+                'type': 'bool32'
+            },
+            {
+                'direction': 'in',
+            	'name': 'timeout',
+                'type': 'float64'
+            },
+            {
+                'direction': 'in',
+                'enum': 'GroupBy',
+	            'name': 'dataLayout',
+                'type': 'int32'
+            },
+            {
+                'direction': 'in',
+                'name': 'writeArray',
+                'type': 'const float64[]'
+            },
+            {
+                'direction': 'out',
+                'name': 'sampsPerChanWritten',
+                'type': 'int32'
+            },
+            {
+                'direction': 'in',
+                'include_in_proto': False,
+	            'name': 'reserved',
+                'pass_null': True,
+                'type': 'bool32',
+            }
+        ],
+        'returns': 'int32'
+    },
+    'WriteDigitalU16': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'task',
+                'type': 'TaskHandle',
+                'grpc_type': 'nidevice_grpc.Session'
+            },
+            {
+                'direction': 'in',
+                'name': 'numSampsPerChan',
+                'type': 'int32',
+            },
+            {
+                'direction': 'in',
+                'name': 'autoStart',
+                'type': 'int32',
+            },
+            {
+                'direction': 'in',
+                'name': 'timeout',
+                'type': 'double',
+            },
+            {
+                'direction': 'in',
+                'name': 'dataLayout',
```

Comment:
Add `'enum': 'GroupBy',`

(Although I'm going to come through and regenerate this, so we don't need to be too fussy)